src/scenes/gamescreen.ts

```diff
@@ -20,15 +44,16 @@ export class GameScreen extends Phaser.Scene {
         this.createWorld();
         this.createCameraDrag();
         this.scene.launch("GameUiScreen");
+        this.placedTilesContainer = this.add.container();
     }
 
+    /**
+     * Temporary: Create three tiles on top of screen that represent the goal tiles\
+     * Need to add starting tile and correct positioning
+     */
     private createWorld(): void {
-        for (let i: int = 0; i < 9; i++) {
-            this.add.image(
-                Math.floor(Math.random() * 8) * 128,
-                Math.floor(Math.random() * 8) * 128,
-                `tile${i}`,
-            );
+        for (let i: int = 0; i < 3; i++) {
+            this.add.image((2 + i * 2) * 128, 0, `tile${8}`);
```

Comment:
why this? what does it do?

Reply:
It displays 3 placeholders for goal tiles on the GameScreen.

Reply:
ah oke i get it. then its fine we can leave this.